src/client/pages/Landing/components/Card.stories.tsx

```diff
@@ -0,0 +1,24 @@
+import React from 'react'
+import { MemoryRouter } from 'react-router-dom'
+import { Card } from './Card'
+
+export default {
+  title: 'Landing Card',
```

Comment:
Maybe group it as a component under Landing? :) 
```suggestion
  title: 'Landing/Card',
```